scripts/signals_assignment.py

```diff
@@ -0,0 +1,24 @@
+# Note: Run this file to dynamically assign the signal numbers to (pyLSV2/pyLSV2/channel_signals.py).
```

Comment:
I don't think this will work the way you intended it. When opening a relativ path, python always starts to search from the location in the command line. Therefor this will only work if the user first changes to the script directory. Otherwise the output file gets written to an arbitrary location.
On the other hand I don't like the idea of writing files to the library folder. Some users might not appreciate this behavior. When I made my earlier suggestion I was thinking more in the direction of writing the file into any folder and for the user to have to import the file into the script themself.
Another approach would be to export the signals to a configuration file instead of python code. pyLSV2 would have two function:
- one for reading the signals from the control and writing them to a config file (e.g. json or xml)
- one to read the config file and create the signal mapping object